src/Illuminate/Foundation/Console/Presets/vue-stubs/ExampleComponent.vue

```diff
@@ -2,7 +2,7 @@
     <div class="container">
         <div class="row justify-content-center">
             <div class="col-md-8">
-                <div class="card">
+                <div class="card card-default">
```

Comment:
Why was this class added? It doesn’t exist in Bootstrap 4.

Reply:
It's currently also added to the example component in the Skeleton but I'll send in prs to remove it from both framework and the skeleton.

Reply:
Good catch btw 👍